Jenkinsfile

```diff
@@ -99,7 +116,7 @@ pipeline {
         versioner(
           token: "${GITHUB_PACKAGES_TOKEN}"
         , dry: true
-        , repo: repo
+        , repo: REPO
```

Comment:
Can you add `, branch: "master"` to the `versioner` blocks? The new version understands that and it'll set us up to change over to using `main` branch whenever we're ready

Reply:
that's still the default for now, right? we can probably just add it when we change the base branch name, unless there's a reason for doing it sooner

Reply:
Just reducing the likelihood it'll be forgotten later. If you don't think it's a prudent course, I'll go with your judgement 😁

Reply:
ah yeah, mainly thinking that if we add it here, we should do it in all of these. I'll follow your lead 👍